src/utils/config.ts

```diff
@@ -108,6 +108,10 @@ export function loadConfig() {
     allow_test_clearing: ENV.ALLOW_TEST_CLEARING || config.allow_test_clearing,
     sql_log: ENV.SQL_LOG || config.sql_log,
     dont_ping_hub: ENV.DONT_PING_HUB || config.dont_ping_hub,
+    transportPrivateKeyLocation:
+      ENV.TRANSPORT_PRIVATE_KEY_LOCATION || config.transportPrivateKeyLocation,
+    transportPublicKeyLocation:
+      ENV.TRANSPORT_PUBLIC_KEY_LOCATION || config.transportPublicKeyLocation,
```

Comment:
Should I create a default value for the location of these new tokens

Reply:
yes, make a default location so existing nodes dont need to update their config file

Reply:
yup updated in these two commits 9bfd2d7 be7ea04